airflow/dev/.env

```diff
@@ -5,4 +5,6 @@ AIRFLOW_CONN_METADATA_DB=postgres+psycopg2://airflow:airflow@postgres:5432/airfl
 AIRFLOW_VAR__METADATA_DB_SCHEMA=airflow
 AIRFLOW__SCHEDULER__SCHEDULER_HEARTBEAT_SEC=10
 AIRFLOW_VAR_INGEST_TO_GCS_SERVICE_ENDPOINT=http://ingest_to_gcs:8080
-AIRFLOW_VAR_GCS_TO_BQ_SERVICE_ENDPOINT=http://gcs_to_bq:8080
\ No newline at end of file
+AIRFLOW_VAR_GCS_TO_BQ_SERVICE_ENDPOINT=http://gcs_to_bq:8080
+# Ensure that the you set CDC_COVID_DEATH_BUCKET with a Cloud Storage bucket name in your executing project
+AIRFLOW_VAR_CDC_COVID_DEATH_BUCKET=${CDC_COVID_DEATH_BUCKET}
```

Comment:
nit: please add a newline at the end of the file

Reply:
done